AK/HashTable.h

```diff
@@ -184,12 +185,16 @@ class HashTable {
     [[nodiscard]] size_t capacity() const { return m_capacity; }
 
     template<typename U, size_t N>
-    void set_from(U (&from_array)[N])
+    bool try_set_from(U (&from_array)[N])
     {
         for (size_t i = 0; i < N; ++i) {
-            set(from_array[i]);
+            if (try_set(from_array[i]) == HashSetResult::Failed)
+                return false;
         }
+        return true;
     }
+    template<typename U, size_t N>
+    void set_from(U (&from_array)[N]) { VERIFY(try_set_from(from_array)); }
```

Comment:
Please don't put side-effects into `VERIFY()` calls, as the calls could get removed altogether.
```suggestion
    void set_from(U (&from_array)[N])
    {
        bool result = try_set_from(from_array);
        VERIFY(result);
    }
```